services/ui-src/src/components/reports/ReportPeriod.tsx

```diff
@@ -0,0 +1,39 @@
+// components
+import { Box, Heading } from "@chakra-ui/react";
+// utils
+import { displayLongformPeriod } from "utils";
+import { AnyObject } from "types";
+
+export const ReportPeriod = ({ text, reportPeriod }: Props) => {
+  const { section, subsection } = text;
+  const retSection = section === "Recruitment, Enrollment, and Transitions";
+  const pageNine = subsection.includes("HCBS");
+  const currentPeriod = displayLongformPeriod(reportPeriod);
+  return (
+    <Box>
+      {retSection &&
+        (!pageNine ? (
+          <Heading as="h2" sx={sx.periodText}>
+            {currentPeriod}
+          </Heading>
+        ) : (
+          <Heading as="h2" sx={sx.periodText}>
+            This page needs a different calculation
```

Comment:
Is this a placeholder for now?

Reply:
yep haha cause Casie said page 9 is its own card